### DISCUSSION TITLE
RDS run failed on XA50

### DISCUSSION THREAD

**Discussion by hyunsoolee0503** [yarra-framework/yarra-framework.github.io#2] [Q&A] [answered]
Recently, one of the vida was upgraded to XA50 so I tried to install the up-to-date (062 ver) yarraclient but failed with a error message " SmartScreen can't be reached right not". I tried with 059_b1 and it worked but I couldn't start the transfer due to the version mismatch. Also, I tried with 060 and got the same error message with 062. I assume the certificates were installed appropriately since it worked with 059_b1. Could you please check this isssue?

**Comment by tblock79** [comment, marked as answer]
Hi!

This is a (known) limitation when running additional software on the XA versions and not directly related to the Yarra Client (so there is nothing we can do about from the Yarra side). Below is a solution that we have obtained and that has solved the problem at our site. However, use at your own risk!

---

SmartScreen is a cloud-based Microsoft service for anti-malware.  I think that if the scanner doesn’t have access to the internet (or more specifically the SmartScreen servers), it won’t work and fails “safe” to block the app. Can you check to see if there are any network settings for the scanner in question that block internet access? Side note for everyone else – I think this might affect WIP installations too when the scanner is without internet access, but didn’t get sufficient information from the colleague who reported the issue to confirm.
 
I think it’s possible to disable SmartScreen via the Group Policy Editor, overriding the “managed by your administrator” below.  

Caveat: This many affect the security of the system and the customer should be the one to make this change if they understand and accept the associated risks:

* In the Start Menu, run “Admin Cmd” and enter the medadmin credentials.
* In the command prompt, run “gpedit”
* Browse to Computer Configuration -> Administrative Templates -> Windows Components -> File Explorer. (Screenshot below)
* Click on “Configure Windows Defender SmartScreen” and select “Enabled”, with the setting “Warn”.
* Click “OK”.  No reboot is necessary.
* Verify the applicable settings in the “App & Browser Control”

I believe that if you allow the app to run once via this “Run Anyways”, then it’ll remember the setting and you may be able to undo the above changes in gpedit.

**Comment by hyunsoolee0503** [comment]
@tblock79 Thank you very much for introducing the workaround! I'll try and get back to you soon.  Additionally, I was just wondering what made 059_b1 different from 059 since 059_b1 worked out but 059 didn't for XA31 with same smartscreen message. I often get the same smartscreen message when I try to install WIP so in that case, I usually unzip the installers and manually copy and paste the relevant files into the folders. In may sites in Korea, the scanners are often disconnected from internet to prevent any malware as far as I know.

**Comment by tblock79** [comment]
Not sure why the popup did not appear for the version 059_b1. There is no significant difference between 059 and 059_b1 that would explain it. I remember that the Smart Defender message started appearing after some service pack was installed on our scanner, so maybe it was just coincidental that the service pack was installed before upgrading to the 059 version.

**Reply by hyunsoolee0503** [reply]
Dear Tobias, I have another question regarding the similar issue. My colleague tried to install yarra RDS on XA50 system and he failed. I told him how to change the smartscreen settings and he changed properly and could push 'run' button after medadmin login. However, after medadmin login, another error message was popped up saying " Your Organization used Windows Defender Application Control to block this app". Do you happen to have any suggestions to solve this issue? Best regards, Hyun-Soo

**Reply by tblock79** [reply]
I believe you need to start the application by right-clicking on the icon and selecting "Run as administrator..." (and then enter the password for medadmin). Otherwise, you will get this error message on XA30 (and newer), as far as I remember.

**Reply by hyunsoolee0503** [reply]
Tobias, thanks for checking the issue. I checked with my colleague and confirmed that he started the application with administration login. However, the windows defender blocked the app as follows.
![1692675777416](https://github.com/yarra-framework/yarra-framework.github.io/assets/61035390/2bc005a9-63a7-4882-9bc2-1e1d66249b87)
